pkg/svcgen/app.go

```diff
@@ -6,7 +6,7 @@ import (
 	"github.com/izumin5210/grapi/pkg/svcgen/params"
 )
 
-type CreateAppFunc func(*gencmd.Ctx, *gencmd.Command) (*App, error)
+type CreateAppFunc func(*gencmd.Command) (*App, error)
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type CreateAppFunc should have comment or be unexported